docs/getting-started/port-your-app.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Running Your Project in Watt
+---
+
+# Running Your Project in Watt
+
+This guide explains how to set up any Node.js project to run in [Watt](https://platformatic.dev/watt), the Node.js Application Server.
+
+## Why Use Watt?
+
+Watt is a powerful environment manager for your Node.js applications that provides:
+
+- Simplified deployment
+- Built-in support for multiple services
+- Multi-threading
+- Standardized environment configuration
+- Service orchestration
+- Gateway capabilities
+- Standardized logging
+- Standardized monitoring
+- ...and much more!
+
+## Setup Instructions
+
+Follow these steps to make your project run in Watt:
+
+## 0. Prerequisites
+Before we begin, make sure you have the following installed:
+
+- [Node.js](https://nodejs.org/) (v20.16.0+ or v22.3.0+)
+- A Node.js application to run. These steps have been tested on the official Fastify [demo](https://github.com/fastify/demo).
+
+### 1. Install Required Dependencies
+
+```bash
+npm install --save wattpm @platformatic/node
+```
+
+### 2. Create Configuration Files
+
+You need two configuration files for Watt:
+
+#### watt.json
+
+This is the main configuration file for Watt. Create it in your project root:
+
+```json
+{
+  "$schema": "https://schemas.platformatic.dev/wattpm/2.63.2.json",
+  "server": {
+    "hostname": "{HOSTNAME}",
+    "port": "{PORT}"
+  },
+  "logger": {
+    "level": "info"
+  },
+  "services": [{
+    "id": "your-service-name",
+    "config": "./watt-svc.json",
+    "path": "."
+  }]
+}
+```
+
+#### watt-svc.json
```

Comment:
I'd probably name this file `watt.service.json` to have continuity with PLT approach and be a little more explicit.
But it's non-blocking advice obviously.

Reply:
Well, but it's not a platformatic service, and I wanted to avoid confusion.